Workbooks/CosmosDb/Cost Optimization/Optimizing Total Provisioned Throughput.workbook

```diff
@@ -0,0 +1,1002 @@
+            },
+            "customWidth": "100",
+            "name": "text - 0"
+          }
+        ]
+      },
+      "name": "group - 7"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "<span style=\"font-size:30px;\"> Background Information </span>\r\n\r\n<br>\r\n<span style = \"font-size:15px;\"> Before we jump into analyzing our workload, there are a few concepts that are important to understand.\r\n## Billing Overview\r\n<span style = \"font-size:15px;\"> In Azure Cosmos DB, you are billed based on your database operations and storage. For database operations, users can choose between manual and autoscale for provisioning throughput. You will be billed based on how many RU/s you provision, regardless of how many you consume. Therefore, selecting the right method of provisioning throughput can help you optimize performance and costs.\r\n<br> \r\n\r\n<span style = \"font-size:15px;\">Manual vs. Autoscale:\r\n-\tManual Provisioned Throughput:  You set the maximum throughput, measured in request units per second (RU/s) required for your workload. The service provisions the capacity needed to support the throughput requirements. You are billed hourly for the maximum provisioned throughput.\r\n-\tAutoscale: You set the highest maximum throughput that you do not want to exceed. Your system automatically scales the throughput T such that 0.1* Tmax <= T <= Tmax. You are only billed for the highest amount of RU/s your system scales every hour. \r\nStorage: User is billed a flat rate for total amount of storage (in GBs) consumed by data and indexes for a given hour. \r\n<br>\r\n\r\n<span style = \"font-size:15px;\">Now that we know how billing works in Azure Cosmos DB and the different choices for provisioning throughput, let’s study how much of our provisioned throughput we are using. As mentioned earlier, you will be billed based on how many RU/s you provision, regardless of how many you consume, so it is important to right-size your workload in order to optimize costs and performance.  \r\n"
```

Comment:
Can we add some links to documentation on learning more about manual v. autoscale